LayoutTests/fast/events/mouse-events-on-textarea-resize.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML>
+<script src="../../resources/js-test.js"></script>
+<style>
+textarea {
+  width: 100px;
+  height: 100px;
+  margin: 10px;
+}
+</style>
+<textarea id="target"></textarea>
+<div id="console"></div>
+<script>
+description("Verifies that correct mouse events are fired and when resizing an element");
+var testEventList = ['focus', 'mousedown', 'mouseup', 'click'];
+var preventDefaultList = ['mousedown', ''];
+var eventToPreventDefault = '';
+function init() {
+  var targetDiv = document.getElementById("target");
```

Comment:
Nit: `target` would be a better name, since this is not a `<div>`.

Reply:
Cool, will do. I think on iOS platform, we might need to add platform specific exception due to "click" event or do you think, I am doing something wrong?